README.md

```diff
@@ -7,9 +7,23 @@ Web dashboard for Docks
 ## Deploying Docks in Production
 [TripleParity/docks](https://github.com/TripleParity/docks) contains the latest instructions for deploying Docks using images from Docker Hub
 
+To deploy the latest version of Docks UI:
+```
+$ docker stack deploy -c docker-compose.yml docks-ui
+```
+
+The version tp be deployed can be changed in the `docker-compose.yml` file.
```

Comment:
Spelling mistake? `tp`?

Reply:
Yes, spelling mistake. Thanks for reading the document :laughing: